Assembus/Assets/Scripts/Models/Project/ProjectSpace.cs

```diff
@@ -10,10 +10,10 @@ public class ProjectSpace
     {
         /// <summary>
         ///     Stores the current numerical index which
-        ///     acts as a postfix for newly created component groups
+        ///     acts as a postfix for newly created component items
```

Comment:
Comment deprecated. Index is no longer used as part of a postfix for groups but rather for a index for names for both groups and components